File: src/petrov/BookListWindow.java
package petrov;

import javax.swing.*;

public class BookListWindow extends JFrame {
    BookListWindow(){
        setDefaultCloseOperation(WindowConstants.DISPOSE_ON_CLOSE);
        setSize(600, 400);
        setLocationRelativeTo(null);
        setResizable(true);//а пусть будет
        setTitle(Set.INSTANCE.getBooks());

        Set.INSTANCE.out("BookList created");
    }//BookList

}

File: src/petrov/BookListWindow.kt
package petrov

import javax.swing.*

class BookListWindow internal constructor() : JFrame() {
    init {
        defaultCloseOperation = WindowConstants.DISPOSE_ON_CLOSE
        setSize(600, 400)
        setLocationRelativeTo(null)
        isResizable = true//а пусть будет
        title = Set.books

        Set.out("BookList created")
    }//BookList

}
